_topic_map.yml

```diff
@@ -206,28 +206,31 @@ Topics:
 - Name: Installing on bare metal
   Dir: installing_bare_metal
   Topics:
-  - Name: Installing a cluster on bare metal
-    File: installing-bare-metal
-  - Name: Installing a cluster on bare metal with network customizations
-    File: installing-bare-metal-network-customizations
-  - Name: Restricted network bare metal installation
-    File: installing-restricted-networks-bare-metal
-- Name: Deploying installer-provisioned clusters on bare metal
-  Dir: installing_bare_metal_ipi
-  Distros: openshift-origin,openshift-enterprise
-  Topics:
-  - Name: Overview
-    File: ipi-install-overview
-  - Name: Prerequisites
-    File: ipi-install-prerequisites
-  - Name: Setting up the environment for an OpenShift installation
-    File: ipi-install-installation-workflow
-  - Name: Post-installation configuration
-    File: ipi-install-post-installation-configuration
-  - Name: Expanding the cluster
-    File: ipi-install-expanding-the-cluster
-  - Name: Troubleshooting
-    File: ipi-install-troubleshooting
+  - Name: Installing user-provisioned clusters on bare metal
+    Dir: installing_bare_metal_upi
+    Topics:
+    - Name: Installing a cluster on bare metal
+      File: installing-bare-metal
+    - Name: Installing a cluster on bare metal with network customizations
+      File: installing-bare-metal-network-customizations
+    - Name: Restricted network bare metal installation
+      File: installing-restricted-networks-bare-metal
+  - Name: Deploying installer-provisioned clusters on bare metal
+    Dir: installing_bare_metal_ipi
+    Distros: openshift-origin,openshift-enterprise
+    Topics:
+    - Name: Overview
+      File: ipi-install-overview
+    - Name: Prerequisites
+      File: ipi-install-prerequisites
+    - Name: Setting up the environment for an OpenShift installation
```

Comment:
From just scanning the TOC, I'm not sure what the difference is between "Prerequisites" and "Setting up the environment for an OpenShift installation." There's also not an explicit assembly for installing a cluster.

Ok, the installation step is pretty deep in "Setting up the environment for an OpenShift installation." I suggest retitling this assembly and [this module](https://deploy-preview-32416--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing_bare_metal_ipi/ipi-install-installation-workflow.html#deploying-the-cluster-via-the-openshift-installer_ipi-install-configuration-files) to talk about installing